IBPSA/BoundaryConditions/SolarGeometry/BaseClasses/SolarAzimuthNoEvent.mo

```diff
@@ -0,0 +1,32 @@
+within IBPSA.BoundaryConditions.SolarGeometry.BaseClasses;
+function SolarAzimuthNoEvent "Determines solar azimuth with no event"
```

Comment:
Use lower case for function names

Reply:
Addressed in https://github.com/ibpsa/modelica-ibpsa/pull/1374/commits/f5d035b7b096c40ac878c4d525be00309bce9265.